scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,19 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    @dec.knownfailureif(True, "Tough NIST Test Cases fail")
```

Comment:
No, I initially thought only of two groups, not failing and failing. 
Or use case specific precision to test worsening of precision compared to current status.